docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
-  - Spec config for CO resources since versions:
-  - 4.6.16+
-  - 4.7+
+### API Reference
+None
+
+### Sample data
+- [docs/insights-archive-sample/pod](./insights-archive-sample/pod)
+- [docs/insights-archive-sample/events](./insights-archive-sample/events)
+
+### Location in archive
+- `config/pod/{namespace}/{pod}.json`
+- `events/`
+- `config/pod/{namespace}/logs/{pod}/{container}_{current|previous}.log`
+
+### Config ID
+`clusterconfig/operators_pods_and_events`
+
+### Released version
+- 4.8.2
+
+### Backported versions
+- 4.6.35+
+- 4.7.11+
+
+### Changes
+- This gather used to be part of [ClusterOperators](#ClusterOperators) before version `4.8.2`.
+- The collected data was previously included as specifications for `ClusterOperators`, and it was initially
+introduced in version `4.3.0` and later backported to version `4.2.10+`.
 
 
 ## ClusterOperators
 
-collects all the ClusterOperators definitions and their related resources
+Collects all the ClusterOperators definitions and their related resources
 from the `operator.openshift.io` group.
 
-The Kubernetes api https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/clusteroperator.go#L62
-Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#clusteroperatorlist-v1config-openshift-io
+### API Reference
+- https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/clusteroperator.go#L62
+- https://docs.openshift.com/container-platform/4.3/rest_api/index.html#clusteroperatorlist-v1config-openshift-io
 
-* Location of operators related resources: config/clusteroperator/{group}/{kind}/{name}
-* Location of operators in archive: config/clusteroperator/
-* Location of operators related resources in older versions: config/clusteroperator/{kind}-{name}
-* See: docs/insights-archive-sample/config/clusteroperator
-* Id in config: clusterconfig/operators
-* Spec config for CO resources since versions:
-  - 4.6.16+
-  - 4.7+
+### Sample data
+- [docs/insights-archive-sample/config/clusteroperator](./insights-archive-sample/config/clusteroperator)
+
+### Location in archive
+| Version   | Path														|
+| --------- | --------------------------------------------------------	|
+| >= 4.2.0  | config/clusteroperator/ 									|
+| < 4.7.0   | config/clusteroperator/{kind}-{name}.json 					|
+| >= 4.7.0  | config/clusteroperator/{group}/{kind}/{name}.json 			|
+| < 4.8.2   | config/pod/{namespace}/{pod}.json							|
+| < 4.8.2   | events/													|
+
+### Config ID
+`clusterconfig/operators`
+
+### Released version
+- 4.2.0
+
+### Backported versions
+None
+
+### Changes
+- `config/pod/{namespace}/{pod}.json` and `events/` were moved to
+[ClusterOperatorPodsAndEvents](#ClusterOperatorPodsAndEvents) since `4.8.2`,
+both were introduced at `4.3.0` as part of this gatherer and backported to `4.2.10+`.
+- ClusterOperator resources was included at `4.7.0` and backported to `4.6.16+`.
```

Comment:
I don't understand what was added/backported. The "Locations" table suggests that the location of the ClusterOperator resources changed in 4.7.0. If the location change was backported to 4.6.16, the "Locations" table is incomplete/inaccurate.